pkg/platform/factory/factory.go

```diff
@@ -148,26 +151,34 @@ func getContainerBuilderConfiguration(platformConfiguration interface{}) *contai
 			"gcr.io/kaniko-project/executor:v0.17.1")
```

Comment:
perhaps we should bump kaniko to `1.18.0` in here as well?

Reply:
We should test the new version
so 👍 but in a separate PR